velox/docs/functions/spark/datetime.rst

```diff
@@ -55,3 +55,7 @@ These functions support TIMESTAMP and DATE input types.
 .. spark:function:: year(x) -> integer
 
     Returns the year from ``x``.
+    
+.. spark:function:: add_months(startDate, numMonths) -> integer
```

Comment:
Functions must appear in alphabetical order.

Reply:
ok, move to head